### PR TITLE
Created a container that uses Redux to store markdown files

### PR DESCRIPTION
**Pull request by caripizza**

### CODE REVIEW

File: src/actions/markdownListActions.test.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { createMarkdown, CREATE_MARKDOWN } from './markdownListActions';
+
+describe('markdown list action tests', () => {
+  it('creates an action that updates the markdown list', () => {
```
Comment (mehtaphysical) [inline src/actions/markdownListActions.test.js]: it appends markdown to a list

File: src/reducers/markdownListReducer.test.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import reducer from './markdownListReducer';
+
+describe('markdown list reducers tests', () => {
+  it('handles an add markdown to markdown list action', () => {
```
Comment (mehtaphysical) [inline src/reducers/markdownListReducer.test.js]: this is a cumbersome and weird sentence to read

**Review by mehtaphysical — APPROVED** [review]
Approved pending tarvis tests

**Review by Zilula — APPROVED** [review]
Looks good Cari!

**Pull request merged**